site/components/Modal/styles.ts

```diff
@@ -74,4 +74,16 @@ export const modalContent = css`
       transform: scale(1);
     }
   }
+
+  //Resolve border radius issues by enforcing rounded corners on scroll bars
+  &::-webkit-scrollbar-thumb {
+    background-color: var(--surface-04);
+    border: 4px solid transparent;
+    border-radius: 8px;
+    background-clip: padding-box;
+  }
+
+  &::-webkit-scrollbar {
+    width: 16px;
```

Comment:
Good one @0xemc :) What browser did you notice the issues on?

Reply:
Just a small comment, we prefer rem units as opposed to px.

Reply:
Cool cool, updated. Do we have a `px to rem` util? Just thinking if the base size ever changes we'd need to go through and update all the rems :( (same for px too).

The scrollbar was strange in Brave (chromium) & Chrome but it's because I'm on Windows 11 I'm fairly sure.

Reply:
we set 1rem to 10px (assuming browser default)
https://github.com/KlimaDAO/klimadao/blob/staging/lib/theme/globals.css#L19

That way if a vision impaired user increases the browser font size, the entire UI should scale comfortably

Reply:
Updated 👍